packages/keyboard-shortcuts/src/store/actions.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Keyboard key combination.
+ *
+ * @typedef {Object} WPShortcutKeyCombination
+ *
+ * @property {string}  character Character.
+ * @property {string?} modifier  Modifier.
+ */
+
+/**
+ * Configuration of a registered keyboard shortcut.
+ *
+ * @typedef {Object} WPShortcutConfig
+ *
+ * @property {string}                      name           Shortcut name.
+ * @property {string}                      category       Shortcut category.
+ * @property {string}                      description    Shortcut description.
+ * @property {WPShortcutKeyCombination}    keyCombination Shortcut key combination.
+ * @property {[WPShortcutKeyCombination]?} aliases        Shortcut aliases.
```

Comment:
If it's meant to be an optional array of combinations:

```suggestion
 * @property {WPShortcutKeyCombination[]} [aliases]        Shortcut aliases.
```

As expressed, this would be typed as a (nullable) array of a single combination.